src/main/java/lotto/LottoMachine.java

```diff
@@ -11,6 +11,11 @@
 
 public class LottoMachine {
 	private static final int LOTTO_PRICE = 1000;
+	private Random random;
```

Comment:
`static`으로 선언 후 사용하면 `LottoMachine` 인스턴스 생성시 매번 `Random` 인스턴스를 생성하지 않아도 됩니다.
```suggestion
	private static final Random random = new Random();
```